kernel/kernel_drivers.go

```diff
@@ -41,6 +41,15 @@ var osSymlink = os.Symlink
 // separated by dots for the kernel version.
 var utsRelease = regexp.MustCompile(`^([0-9]+\.){2}[0-9]+`)
 
+const earlyKernelsDir = "/run/mnt/kernel-snaps"
+
+// EarlyKernelModsCompMountDir returns the path where components are mounted on
+// early boot.
+func EarlyKernelModsCompMountDir(compName string, compRev snap.Revision, ksnapName string, snapRev snap.Revision) string {
```

Comment:
nitpick: as discussed, it's a bit odd this doesn't contain GlobalRootDir but not a blocker

Reply:
Changed, I was reluctant because it gets problematic in the test when mount units are created, but I'll wait until we use the directory for that.